src/data-workspace/data-entry-cell/use-data-value-params.js

```diff
@@ -0,0 +1,44 @@
+import { useContextSelection } from '../../context-selection/index.js'
+import { useMetadata } from '../../metadata/index.js'
+import {
+    getCategoryComboById,
+    getDataSetById,
+} from '../../metadata/selectors.js'
+
+export const useDataValueParams = ({ deId, cocId }) => {
+    const [dataEntryContext] = useContextSelection()
+    const metadataFetch = useMetadata()
+
+    if (metadataFetch.isLoading || metadataFetch.isError) {
+        return null
+    }
+
+    const { dataSetId, orgUnitId, periodId, attributeOptionComboSelection } =
+        dataEntryContext
+
+    const attributeComboId = getDataSetById(metadataFetch.data, dataSetId)
+        .categoryCombo.id
+    const isDefaultAttributeCombo = getCategoryComboById(
+        metadataFetch.data,
+        attributeComboId
+    ).isDefault
+
+    const dvParams = {
```

Comment:
Personal opinion: I'm against abbreviations. It just saves a bit if writing time l, but that shouldn't matter. It makes code harder to read though, so I don't think we should do it unless we're dealing with very long names